block-layer/graphsync/graphsync.md

```diff
@@ -59,6 +59,99 @@ type GraphSyncNet interface {
 
 ## Network Messages
 
+### Protocol Version 1.1.0
+
+Graphsync network messages are encoded in DAG-CBOR. They have the following schema
+
+```ipldsch
+type GraphSyncExtensions {string:Any}
+type GraphSyncRequestID int
+type GraphSyncPriority int
+
+type GraphSyncMetadatum struct {
+  Link         &Any
+	BlockPresent bool
```

Comment:
s/bool/Bool/

The rule is based on whether you're refering to a kind (which is a keyword) or a type (which are by convention TitleCase).  (We're not quite the same as golang here, and don't have support them interchangably.)

`type Foo int` is a declaration using a kind keyword, so it's lowercase.

`type Foo struct { Field Bool }` is refering to a type, so it's using a name.

(There's an implicit prelude in a schema which defines `type Bool bool`, `type String string`, etc, so those are all available as a type name to make this less arduous.)